time.go

```diff
@@ -134,6 +136,21 @@ func (t Time) IsBankingDay() bool {
 	return true
 }
 
+// AddBusinessDay takes an integer for the number of valid business days to add and returns a Time.
+// Negative values and large values (over 500 days) will not modify the Time.
+func (t Time) AddBusinessDay(d int) Time {
+	if d < 1 || d > 500 {
```

Comment:
Why 500?  Should that be configurable, or maybe a constant?

Reply:
It looks like a limit to avoid an overly large calculation.